### PR TITLE
[IMP] Add loaders

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

**Review by simahawk — APPROVED** [review]
FTR I remember that w/ @guewen w/ discussed if we should move these helpers from our project's template and we agree to not pollute anthem w/ all sorts of loaders variations.

To me is fine anyway (and I'm glad we wrote all those comments :smile: ).

File: anthem/lyrics/loaders.py [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
+    """ Use to load warehouses in multi-company
+
+    In multicompany mode we must force the company otherwise the sequences that stock
+    module generates automatically will have the wrong company assigned.
+
+    Usage::
+
+        @anthem.log
+        def import_warehouses(ctx):
+            load_warehouses(ctx, 'stock.warehouse.csv')
```
Comment (yvaucher) [inline anthem/lyrics/loaders.py:53]: The company parameter is missing in "Usage".

**Review by yvaucher — APPROVED** [review]
LGTM

Just a little docstring to fix

File: anthem/lyrics/loaders.py (hunk 1 above)
Comment (gurneyalex) [reply anthem/lyrics/loaders.py:53]: good catch

**Comment by max3903** [comment]
> FTR I remember that w/ @guewen w/ discussed if we should move these helpers from our project's template and we agree to not pollute anthem w/ all sorts of loaders variations.
> 
> To me is fine anyway (and I'm glad we wrote all those comments  ).

File: anthem/lyrics/modules.py [hunk 2]
```diff
@@ -8,35 +8,37 @@
...
     if not module_list:
-        raise AnthemError(u"You have to provide a list of "
-                          "module's name to uninstall")
+        raise AnthemError(
+            u"You have to provide a list of module's name to uninstall"
+        )
 
-    mods = ctx.env['ir.module.module'].search([('name', 'in', module_list)])
+    mods = ctx.env["ir.module.module"].search([
+        ("name", "in", module_list), ("state", "!=", "uninstalled")
```
Comment (max3903) [inline anthem/lyrics/modules.py:16]: FYI Skipping the uninstalled modules. Useful when running with `MARABUNTA_FORCE_VERSION`

CC: @JevinD

File: .gitignore [hunk 3]
```diff
@@ -96,3 +96,6 @@ odoo.tar.gz
 odoo-9.0
 odoo-10.0
 odoo-11.0
+
```
Comment (simahawk) [inline .gitignore:99]: this should stay in your global .gitignore :wink:

**Review by simahawk — COMMENTED** [review]
hmm @max3093 you are running black but there's no black config here. Could you apply this change in another PR?

**Review by simahawk — CHANGES_REQUESTED** [review]
blocking till we get to the initial state of the PR that was approved

File: .gitignore (hunk 3 above)
Comment (max3903) [reply .gitignore]: :+1: I learnt something today... Thanks!

**Comment by max3903** [comment]
> blocking till we get to the initial state of the PR that was approved

Fixed!

**Pull request merged**